src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+
+  &__logo {
+    line-height: 0;
+  }
+
+  .nav {
+    &__list {
+      list-style: none;
+      width: max-content;
+      display: flex;
+    }
+
+    &__link {
+      text-decoration: none;
+      color: #{$main-accent-color};
+      text-transform: uppercase;
+      height: 60px;
+      display: flex;
+      position: relative;
+      font-size: 12px;
+      font-weight: 500;
+      align-items: center;
+    }
+
+    &__link:hover {
+      color: #{$blue-color};
+    }
+
+    &__link.is-active {
+      color: #{$blue-color};
+    }
+
+    &__link.is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      width: 100%;
+      height: 4px;
+      background-color: #{$blue-color};
+      border-radius: 8px;
+    }
```

Comment:
You can place it under `&__link`. It would be better due to sass nesting capabilities